src/Promitor.Integrations.Sinks.OpenTelemetry/Configuration/OpenTelemetryCollectorSinkConfiguration.cs

```diff
@@ -1,7 +1,11 @@
-﻿namespace Promitor.Integrations.Sinks.OpenTelemetry.Configuration
+﻿using OpenTelemetry.Exporter;
+
+namespace Promitor.Integrations.Sinks.OpenTelemetry.Configuration
 {
     public class OpenTelemetryCollectorSinkConfiguration
     {
+        public OtlpExportProtocol Protocol { get; set; } = default(OtlpExportProtocol);
```

Comment:
What is the default? THinking of this again, we need to make sure we don't break existing users

Reply:
Thinking of it again, this is probably what we want:
```suggestion
        public OtlpExportProtocol? Protocol { get; set; };
```

Then in the code we will check if it was specified. If it was not, use the current protocol.